06-data-subsetting.md

```diff
@@ -1318,3 +1331,29 @@ be changed with the third argument, `drop = FALSE`).
 > gapminder_small <- gapminder[c(1:9, 19:23),]
 > ~~~
 >
+
+> ## Solution to challenge 7 {.challenge}
+>
+> How many countries in Asia had a population in either 2002 or 2007 above 20 million? 
+> Tip: Use subsetting to maintain the relevant data and then use the function `unique()` to remove duplicate countries.
+>
+> A. 19
+>
+> B. 20
+>
+> C. 21
+>
+> D. 22
+>
+> Answer: B
+> 
+> The answer can be derived using the following statement
+> ~~~{.r}
+> length(unique(
+>   gapminder[
+>     gapminder$year >= 2002 & 
+>     gapminder$year <= 2007 & 
+>     gapminder$continent=='Asia' & 
```

Comment:
I think style guides suggest a space on either side of the `==` in these expressions.